### PR TITLE
Get friend list

### LINKED ISSUE #32: Author 012-b GET List of Current Friends
GET request to return current list of friends 
refer to API specs.

### PR DESCRIPTION
**Pull request by hiufungk**
resolve #32 
added functionality to get list of friends
fixed author profile argument from url to be escaped

### CODE REVIEW

File: backend/api/serializers.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ class Meta:
             'bio'
         )
 
+
```
Comment (forgeno) [inline backend/api/serializers.py:64]: remove this empty line

File: backend/api/view/AuthorProfileView.py [hunk 2]
```diff
@@ -25,10 +25,13 @@ def get(self, request, uid):
         if(authorId == ""):
             return Response("Error: Author ID required!", status.HTTP_400_BAD_REQUEST)
 
-        query_set = AuthorProfile.objects.filter(id=authorId)
-        
-        if (len(query_set) == 1):
-            response_data = AuthorProfileSerializer(query_set[0]).data
-            return Response(response_data, status.HTTP_200_OK)
```
Comment (forgeno) [inline backend/api/view/AuthorProfileView.py:32]: Why was this removed? Don't we need one for if length == 1?

**Review by forgeno — CHANGES_REQUESTED** [review]

**Review by forgeno — APPROVED** [review]

**Review by htruong4 — APPROVED** [review]
